dln.c

```diff
@@ -531,3 +531,27 @@ dln_load(const char *file)
 
     return 0;			/* dummy return */
 }
+
+#define DESTRUCT_FUNC_PREFIX "Destruct_"
```

Comment:
```suggestion
#define DESTRUCT_FUNC_PREFIX EXTERNAL_PREFIX"Destruct_"
```

Reply:
And on many Unix-like systems, the opposite of `init` is `fini` historically.

Reply:
I'm open to calling it Fini as well.